src/templates/tag.js

```diff
@@ -112,7 +122,7 @@ const Tag = props => {
                             <AuthorByline>
                                 <SyledAuthorImage image={author_image} />
                                 <AuthorName>
-                                    <H2>{name}</H2>
+                                    <H1>{name}</H1>
```

Comment:
We also want to add the pagination into here as well like we are doing for the `title` in the other PR. Can we add that in here please?